modules/contact/test/tests/simple_contact/two_block_compress/tests

```diff
@@ -11,6 +11,17 @@
     min_ad_size = 100
     abs_zero = 1.0e-8
   []
+  [frictionless-compress-2d_pg]
+    type = 'CSVDiff'
+    input = 'two_equal_blocks_compress_2d_pg.i'
+    csvdiff = 'two_equal_blocks_compress_2d_pg_out.csv'
+    mesh_mode = 'REPLICATED'
+    requirement = 'The system shall simulate correct contact behavior in 2D when two blocks with the '
+                  'same height come into contact using the Petrov-Galerkin approach'
+    min_ad_size = 100
+    abs_zero = 1.0e-5
+    rel_err = 1e-3
```

Comment:
This seems quite large. Is there a reason it needs to be this large?

Reply:
Oh this was due to my initial attempt to try to match the numbers from the case without PG. I have updated the gold file and removed this loose tolerance.